mutiny-core/src/nodemanager.rs

```diff
@@ -1638,14 +1640,37 @@ impl<S: MutinyStorage> NodeManager<S> {
         from_node: &PublicKey,
         lnurl: &LnUrl,
         amount_sats: u64,
+        zap_npub: Option<XOnlyPublicKey>,
```

Comment:
which npub is this? the wallet owner or the reciever? And how should it be retrieved?

Reply:
the receiver, I figured we'd just be pulling it from the contacts list

Reply:
is this somehow not combined with lnurl in a way? I thought zaps were just an lnurl extension and the receiver pubkey was part of that somehow.

Reply:
No, in the lnurl response it has the pubkey of the key that will sign the zap receipt, but that is just so you can verify that it is not a fake zap